crates/sui-types/src/messages_grpc.rs

```diff
@@ -164,6 +164,11 @@ pub struct HandleCertificateResponseV2 {
     pub fastpath_input_objects: Vec<Object>,
 }
 
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct HandleSoftBundleCertificatesResponseV2 {
+    pub responses: Vec<HandleCertificateResponseV2>,
```

Comment:
Just FYI there is another pending change on grpc interfaces: https://github.com/MystenLabs/sui/pull/17283
cc @bmwill